packages/augur-ui/src/modules/portfolio/components/common/filter-box.tsx

```diff
@@ -167,13 +171,34 @@ const FilterBox: React.FC<FilterBoxProps> = props => {
       hide={hide}
       extend={extend}
       bottomBarContent={
-        <SwitchLabelsGroup
-          tabs={tabs}
-          selectedTab={selectedTab}
-          selectTab={(tab) => {
-            setSelectedTab(tab)
-          }}
-        />
+        getTheme() === THEMES.TRADING ?
+          <SwitchLabelsGroup
+            tabs={tabs}
+            selectedTab={selectedTab}
+            selectTab={(tab) => {
+              console.log('selectTab', tab)
+              setSelectedTab(tab)
+            }}
+          />
+          :
+          <div>
+            <span>Status:</span>
+            <SquareDropdown
+              defaultValue={selectedTab}
+              options={tabs}
+              onChange={(tab) => {
+                console.log('onChange', tab);
```

Comment:
don't forget to remove the console logs in this and the above action in `SwitchLabelGroup`